integration-tests/features/stack_analyses.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Jobs API
+
+  Scenario: Check that the API entry point
+    Given System is running
+    When I access /api/stack-analyses
+    Then I should get 404 status code
+
+  Scenario: Check that the API entry point
+    Given System is running
+    When I wait 10 seconds
+    Then I wait 10 seconds
```

Comment:
Well the problem is that component-analyses and stack-analyses endpoints are ready to accept API calls after some time, even when the system seems to be started properly. I talked about this issue with @fridex and ATM it seems there's no proper way how to ensure that the queue/workers will accept commands via API calls. So this is quick & dirty (and ugly as well) 1st attempt how to let the system to take a breath ;) I'm going to propose slight API design change to do such tests properly.